docs/Analysis_Engine.md

```diff
@@ -92,4 +198,6 @@ Explanation of fields:
       * `utilityLcb` - The LCB of the move's utility.
       * `order` - KataGo's ranking of the move. 0 is the best, 1 is the next best, and so on.
       * `pv` - The principal variation following this move. May be of variable length or even empty.
+   * `rootInfo`: A JSON dictionary with fields containing statistics for requested move itself calculated in the same way as they would be for the next moves. Current fields are: `winrate`, `scoreLead`, `scoreSelfplay`, `utility`, `lcb`, `utilityLcb`, `visits`.
```

Comment:
Should this also be optional? Or is it highly unlikely for json parsers and usage to break if this field suddenly un-optionally pops up in the analysis output?